packages/flutter_tools/templates/plugin/android-kotlin.tmpl/src/main/kotlin/androidIdentifier/pluginClass.kt.tmpl

```diff
@@ -1,5 +1,29 @@
 package {{androidIdentifier}}
 
+{{#useNewAndroidEmbedding}}
+import io.flutter.embedding.engine.plugins.FlutterPlugin
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler
+import io.flutter.plugin.common.MethodChannel.Result
+
+/** {{pluginClass}} */
+public class {{pluginClass}}: FlutterPlugin, MethodCallHandler {
```

Comment:
Same comments here about `@NonNull` annotations and separating `MethodCallHandler`